01-node-tutorial/answers/02-globals.js

```diff
@@ -0,0 +1,5 @@
+console.log("Dir: " + __dirname);
```

Comment:
This is great! Nothing wrong with this syntax whatsoever but but just something to chew on here - console logs can take all kinds of args.- objects, arrays, functions or multiple variables. I sometimes find it easier in a case like this to just psas 2 args like this `console.log("Dir: ", __dirname)`. (all just a matter of preference though)

Reply:
This way is better than string concatenation. Thank you!